src/controllers/hid/hidcontroller.h

```diff
@@ -71,32 +71,64 @@ class HidControllerJSProxy : public ControllerJSProxy {
     /// @param dataList Data to send as list of bytes
     /// @param length Unused but mandatory argument
     /// @param reportID 1...255 for HID devices that uses ReportIDs - or 0 for devices, which don't use ReportIDs
-    /// @param resendUnchangedReport If set, the report will also be send, if the data are unchanged since last sending
+    /// @param useNonSkippingFIFO (optional)
```

Comment:
How about using an `enum class SendReportDispatchStrategy { LatestDedup, Fifo }`? Documenting the enum variants once instead of duplicating the documentation for every function would be more suitable.

Reply:
AFAIK, it's not possible to access C++ enums from JavaScript code, or am I wrong here?

Reply:
I didn't consider that the signature must be compatible with JS. But simply duplicating the documentation is not maintainable.

Reply:
Or use constants (int or bool) and document those instead.